contribs/src/main/java/com/netflix/conductor/contribs/publisher/RestClientManager.java

```diff
@@ -38,6 +26,7 @@ the Connection Manager Timeout (http.connection-manager.timeout) – the time to
         TASK,
         WORKFLOW
     };
+
     private static final String TASK_NOTIFICATION_ENDPOINT = "workflow/TaskNotifications";
     private static final String WORKFLOW_NOTIFICATION_ENDPOINT = "workflow/WorkflowNotifications";
     private static final String URL = "http://bullwinkle.default.svc.cluster.local:7979/v1";
```

Comment:
this URL is specific to Cisco.  It's better to keep this in the configuration file.

Reply:
Done